frontend/src/services/agent.js

```diff
@@ -23,16 +23,10 @@ const client = createClient();
 const clientWithoutErrorManagement = createClient();
 
 client.interceptors.response.use(
-  response => {
-    if (!response.data.success) {
-      store.commonStore.setNotification(response.data);
-      throw new Error(response.data.message);
-    }
-    return response.data;
-  },
+  response => response.data,
   error => {
     if (error.response) {
-      store.commonStore.setNotification(error.response.data);
+      store.commonStore.setNotification(error.response.data, false);
```

Comment:
Won't we display server error (real errors not `CustomError`) here? I didn't saw anything catching all unexpected errors to display a nice message like "Oops, something failed" but we may have already done it.

Reply:
we have a filter in the errorManager middleware that takes care of that
it displays the 500 error in dev but a generic message in prod